tests/integration/components/column-chart-test.js

```diff
@@ -135,3 +135,194 @@ module('Integration | Component | column chart', function (hooks) {
     });
 });
+
+// collision detection test parameters
+const collisionTestParameters = function () {
+    return {
+        type: 'GROUPED',
+
+        series: [
+            {
+                title: 'Total Fruit Eaten'
+            }
+        ],
+
+        xAxis: {
+            domain: [
+                moment('2016-11-01'),
+                moment('2016-11-07')
+            ],
+            ticks: 5
+        },
+
+        yAxis: {
+            ticks: 3,
+            domain: [5, 55]
+        },
+
+        labelOptions: {
+            showMaxMin: true,
+            showDataValues: true,
+            labelCollisionResolution: 'auto'
+        }
+    };
+};
+
+const rawData = [
+    {
+        date: new Date('2016-11-02'),
+        fruits: 10
+    },
+    {
+        date: new Date('2016-11-03'),
+        fruits: 20
+    },
+    {
+        date: new Date('2016-11-04'),
+        fruits: 30
+    },
+    {
+        date: new Date('2016-11-05'),
+        fruits: 40
+    },
+    {
+        date: new Date('2016-11-06'),
+        fruits: 50
+    }
+];
+
+const createDimensionAndGroup = (data) => {
+    const dimension = crossfilter(data).dimension(d => d.date);
+    const group = [dimension.group().reduceSum(item => item.fruits)];
+    return {
+        dimension,
+        group
+    };
+};
+
+module('Integration | Component | column chart - collision', function (hooks) {
+    setupRenderingTest(hooks);
+    hooks.beforeEach(function () {
+        this.set('params', collisionTestParameters());
+        this.owner.register('service:resizeDetector', Service.extend({
+            setup(elementId, callback) {
+                callback();
+            },
+            teardown() {}
+        }));
+    });
+
+    test('removes any regular label that overlaps max from the left', async function (assert) {
+        rawData[3].fruits = 40.23232323232323;
+        const dimAndGrp = createDimensionAndGroup(rawData);
+        this.set('params.dimension', dimAndGrp.dimension);
+        this.set('params.group', dimAndGrp.group);
+        await render(
+            hbs `{{column-chart
+                    dimension=params.dimension
+                    group=params.group
+                    type=params.type
+                    seriesMaxMin=0
+                    series=params.series
+                    xAxis=params.xAxis
+                    yAxis=params.yAxis
+                    labelOptions=params.labelOptions
+                    height=200
+                    instantRun=true
+                }}`);
+        assert.equal([...document.querySelectorAll('.data-text')].map(d => d.innerHTML).indexOf('40.23232323232323'), -1, 'Expected chart does not have regular label if it overlaps max.');
+        assert.dom('.data-text').exists({ count: 2 }, 'Expected 5 data values to be rendered');
+    });
+
+    test('removes any regular label if max overlaps that from the left', async function (assert) {
```

Comment:
Looks like the same test string as the previous one. Is it intended to be `from the right`?

Reply:
the texts are different, 
previous one :  regular label overlaps max from the left
this one: max label overlaps regular label from the left,

will change the string to make it clearer.